src/server.c

```diff
@@ -5866,17 +4061,73 @@ void addReplyCommand(client *c, struct redisCommand *cmd) {
                 lastkey += firstkey;
             keystep = cmd->legacy_range_key_spec.fk.range.keystep;
         }
-        /* We are adding: command name, arg count, flags, first, last, offset, categories, key args, subcommands */
-        addReplyArrayLen(c, 9);
+
+        /* We are adding: command name, arg count, flags, first, last, offset, categories, additional information (map) */
+        addReplyArrayLen(c, 8);
```

Comment:
let's mention this fact in the top comment where we describe the interface changes (i.e. the mix of array of fixed indexes, and a map).

p.s. we'll need a big redis-doc PR to describe all of that, and in theory maybe we wanna avoid merging this PR until the doc is ready.
not doing so can lead to two things:
1. we'll forget or run out of time, and release this undocumented.
2. we'll get some new realizations when documenting it about the usefulness or clarity of something and it may be too late to fix.
3. it may be easier for other reviewers to review the docs rather than the code.

please add a todo bullet in the top comment for docs.